dbt-cta/mobilize/models/1_cta_full_refresh/van_shifts_base.sql

```diff
@@ -10,7 +10,7 @@
     tags = [ "top-level" ]
 ) }}
 -- Final base SQL model
--- depends_on: __dbt__cte___airbyte_org_7e6abb14a314439581d38bb69eaa0083_van_shifts_ab3
+-- depends_on: __dbt__cte___airbyte_org_7e6abb14a314439581d38bb69eaa0083_van_shifts_ab4
```

Comment:
```suggestion
-- depends_on: {{ ref('van_shifts_ab4') }}
```

lol idk what happened in the original version of this file - guess it wasn't actually breaking anything, but we might as well fix this